src/exporter.js

```diff
@@ -0,0 +1,174 @@
+const sha1 = require('sha1');
+
+export default class {
+  constructor(deckName) {
```

Comment:
Add the second "options" argument as in https://github.com/ewnd9/anki-apkg-export/pull/7#issuecomment-263780460

```js
export default function(deckName) {
  return new ApkgExporter(deckName, { sql, template });
}
```